plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptionsVariationSelectorAttributeOptions.php

```diff
@@ -16,11 +18,203 @@ class AddToCartWithOptionsVariationSelectorAttributeOptions extends AbstractBloc
 	protected $block_name = 'add-to-cart-with-options-variation-selector-attribute-options';
 
 	/**
-	 * Get the frontend style handle for this block type.
+	 * Get the block's attributes.
+	 *
+	 * @param array $attributes Block attributes. Default empty array.
+	 * @return array  Block attributes merged with defaults.
+	 */
+	private function parse_attributes( $attributes ) {
+		// These should match what's set in JS `registerBlockType`.
+		$defaults = array(
+			'style' => 'pills',
+		);
+
+		return wp_parse_args( $attributes, $defaults );
+	}
+
+	/**
+	 * Render the block.
+	 *
+	 * @param array    $attributes Block attributes.
+	 * @param string   $content Block content.
+	 * @param WP_Block $block Block instance.
+	 * @return string Rendered block output.
+	 */
+	protected function render( $attributes, $content, $block ): string {
+		global $product;
+		global $attribute_name;
+		global $attribute_terms;
```

Comment:
`global $attribute_terms;` doesn't seem to be used in this method, no?

Reply:
done